### DISCUSSION TITLE
Telescope open file in vsplit

### DISCUSSION THREAD

**Discussion by MijikHna** [nvim-tree/nvim-tree.lua#2204] [Q&A] [answered]
If I open a file in **telescope** with `Ctrl+v` it opens aa file in vertical split to the left side of the **nvim-tree**. Is there any setting that will keep **nvim-tree** buffer always on the left side. For example if I open a file within **nvim-tree** buffer  with `Ctrl+v` it opens the file properly to the right side of the **nvim-tree**.

**Comment by alex-courtis** [comment, marked as answer]
Unfortunately we cannot influence the behaviour of other plugins.

You could create a telescope mapping that positions the split window in your desired location.

`:help telescope.mappings`

**Reply by MijikHna** [reply]
Thanks. At the moment I have ended up with setting `splitright`.  At first glance I haven't found options for `select_vertical` in telescope
